### PR TITLE
Adding Actors API routes and controllers

### PR DESCRIPTION
**Pull request by melonrush13**
Adding actors.ts files to routes and controllers to filter actors by id, and grab all actors.

### CODE REVIEW

File: src/app/controllers/actor.ts [hunk 1]
```diff
@@ -4,23 +4,71 @@ import { collection, database } from "../../db/dbconstants";
...
 /**
- * returns all genres from cosmos db instance
+ * Retrieve and Returns all actors
  * @param req request object
  * @param res response object
  */
 export async function getAll(req, res) {
 
-    telemetryClient.trackEvent({name: "getAll endpoint"});
+    telemetryClient.trackEvent({name: "getAll actors endpoint"});
```
Comment (frodopwns) [inline src/app/controllers/actor.ts]: Looks like you may be working with a slightly old version of master. Take a look at how thiws is being done in master here: https://github.com/Microsoft/helium/blob/master/src/app/controllers/movie.ts#L15

Might need to rebase. We can look at that tomorrow.

Comment (frodopwns) [reply src/app/controllers/actor.ts]: Also

```suggestion
    telemetryClient.trackEvent({name: "get all actors"});
```

File: src/db/dbconstants.ts [hunk 2]
```diff
@@ -1,2 +1,3 @@
 export const database = "imdb";
 export const collection = "movies";
+export const api = "/api";
```
Comment (frodopwns) [inline src/db/dbconstants.ts]: We will want to create or use a constants file more global than this one. The `db` package should only contain values required by the db client.

Comment (melonrush13) [reply src/db/dbconstants.ts]: fixed and create a new file 'constants.ts' under config

**Pull request closed without merging**